### PR TITLE
Staff Directory: Sprint 4 deployment

### PR DESCRIPTION
**Pull request by z3cka**
Stuff in sprint 4: https://jira.library.ucla.edu/secure/RapidBoard.jspa?rapidView=31

@akohler just a quick glance needed to see if I missed anything.

### CODE REVIEW

**Comment by z3cka** [comment]
deploying to `www-test` now

File: www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module [hunk 1]
```diff
@@ -6,6 +6,38 @@
...
+function uclalib_staff_directory_page_alter(&$page) {
+  if ((arg(0) == 'node') && (arg(2) == 'staff')) {
+    // Generate a staff alias if it does not exist
+    $request_uri = request_uri();
+    $alias = drupal_get_path_alias();
+    $path = drupal_get_normal_path($alias);
+    // detect if an alias needs to be created
+    if ($path == $alias) {
+      $location_alias = drupal_get_path_alias('node/'. arg(1));
+      dpm($location_alias, "\$location_alias");
```
Comment (akohler) [inline www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module]: TODO: DPM needs to be removed for production

File: www/sites/all/themes/uclalib_omega/sass/components/_events-teaser.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@
   max-width: 287px;
   text-align: center;
 
-  // Images only shown on homepage ¯\_(ツ)_/¯
```
Comment (akohler) [inline www/sites/all/themes/uclalib_omega/sass/components/_events-teaser.scss]: Awwww, shame to lose shrugging man!

File: www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module (hunk 1 above)
Comment (z3cka) [reply www/sites/all/modules/features/uclalib_staff_directory/uclalib_staff_directory.module]: oops! good catch, will remove!

File: www/sites/all/themes/uclalib_omega/sass/components/_events-teaser.scss (hunk 2 above)
Comment (z3cka) [reply www/sites/all/themes/uclalib_omega/sass/components/_events-teaser.scss]: ya, my SASS compiler did like the `ツ` character ... go figure, never complained before that, and I don't have the time/energy to resolve this at this time.

**Pull request merged**